skopt/tests/test_common.py

```diff
@@ -197,12 +197,12 @@ def test_minimizer_with_space(minimizer):
 
 @pytest.mark.slow_test
 @pytest.mark.parametrize("n_random_starts, optimizer_func",
-                         product([0, 5],
+                         product([0, 1, 2, 3, 4, 5],
                                  [gp_minimize,
                                   forest_minimize,
                                   gbrt_minimize]))
 def test_init_vals_and_models(n_random_starts, optimizer_func):
-    # test how many models are fitted when using initial points, values
+    # test how many models are fitted when using initial points, y0 values
```

Comment:
Maybe a bit more clear would be "initial points and corresponding objectives", otherwise this might be confusing to someone. Also maybe something like "different number of initial random starts"
Edited: saw the other test.